src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageActionInvokerCacheEntry.cs

```diff
@@ -53,6 +56,8 @@ public class PageActionInvokerCacheEntry
         /// </summary>
         public Func<Page, object, Task> PropertyBinder { get; }
 
+        public ParameterBinder ParameterBinder { get; }
```

Comment:
This doesn't need to be part of the CacheEntry, it's a service https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvokerProvider.cs#L30